buildSrc/src/main/groovy/io.micronaut.build.internal.data-kotlin.gradle

```diff
@@ -18,3 +18,16 @@ tasks.named("compileTestKotlin") {
         jvmTarget = '17'
     }
 }
+
+kotlin {
```

Comment:
@wetted see this possible workaround for Kapt + 21

Reply:
Thanks, I already pushed that change for `micronaut-object-storage` but still have issues. I'll look into it again Monday morning.